driver-tests/shading/shaded/src/test/java/com/datastax/driver/core/NettyUtilTest.java

```diff
@@ -0,0 +1,29 @@
+package com.datastax.driver.core;
+
+/*
+ * Copyright (C) 2012-2017 DataStax Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class NettyUtilTest {
+
+    @Test(groups = "short")
```

Comment:
awesome, nice and straightforward!  Can you change the group to `unit` for this and the other test?   Since it has no external dependencies i think it's ok to be a unit test.

Reply:
Yep, just changing it to unit. I was not sure if "unit" is the right group for tests which run after the assembly (which happens here - it's not really a unit but an integration test).

But no problem to change it to unit here if you want.